testsuite/features/secondary/min_docker_api.feature

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2017-2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@scope_building_container_images
+Feature: API "image" namespace for containers and sub-namespaces
+
+  Scenario: Test "image.store" namespace
+    When I am logged in API as user "admin" and password "admin"
+    And I create and delete an image store via API
+    And I list image store types and image stores via API
+    And I set and get details of image store via API
+
+  Scenario: Test "image.profiles" namespace
+    When I create and delete profiles via API
+    And I create and delete profile custom values via API
+    And I list image profiles via API
+    And I set and get profile details via API
+    And I logout from API
+
+# Note: image building via API is not tested here
+#       it is tested as another feature
```

Comment:
Can we move this comment in the top of the file and then be specific with the filename/path of the other feature?

Reply:
ok